{"uid":"1eef9383400a3e2f","name":"Verify if the user redirect to linkedin","fullName":"Validate Contact with US cards Redirection.Verify if the user redirect to linkedin","historyId":"c81908d56dbcd079b877da899a5bc402","time":{"start":1672262084551,"stop":1672262092406,"duration":7855},"description":"","descriptionHtml":"","status":"failed","statusMessage":"Timed out retrying after 4000ms: expected '' to include 'company/appiskey/'\n                \n                The command was expected to run against origin `https://qa.appiskey.com` but the application is at origin `https://www.linkedin.com`.\n                \n                This commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly.","statusTrace":"AssertionError: Timed out retrying after 4000ms: expected '' to include 'company/appiskey/'\n                \n                The command was expected to run against origin `https://qa.appiskey.com` but the application is at origin `https://www.linkedin.com`.\n                \n                This commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly.\n                    at Context.eval (webpack:///./cypress/e2e/testcases/Services.cy.js:65:17)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"Validate Contact with US cards Redirection"},{"name":"testClass","value":"Validate Contact with US cards Redirection"},{"name":"testMethod","value":"Verify if the user redirect to linkedin"},{"name":"package","value":"Validate Contact with US cards Redirection"},{"name":"resultFormat","value":"allure1"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"1eef9383400a3e2f.json","parameterValues":[]}